soc/nxp/mcx/mcxw/soc.c

```diff
@@ -119,6 +119,8 @@ static ALWAYS_INLINE void clock_init(void)
 	CLOCK_SetIpSrcDiv(kCLOCK_Lpi2c0, kSCG_SysClkDivBy16);
 	CLOCK_SetIpSrc(kCLOCK_Lpi2c1, kCLOCK_IpSrcFro192M);
 	CLOCK_SetIpSrcDiv(kCLOCK_Lpi2c1, kSCG_SysClkDivBy16);
+	CLOCK_SetIpSrc(kCLOCK_Lpadc0, kCLOCK_IpSrcFro192M);
```

Comment:
Can we keep this clocking within the `if (DT_NODE_HAS_COMPAT_STATUS(adc0, nxp_lpadc, okay))` block, or is it needed for multiple IP blocks?

Reply:
I am just following the precedent in the file